examples/hybrid/TurbulenceConvectionUtils.jl

```diff
@@ -145,14 +145,19 @@ function init_tc!(Y, p, params, colidx)
             surf_ref_thermo_state,
         )
     else
-        @. p.ᶜp[colidx] = p.edmf_cache.ᶜp₀
-
-        CA.compute_ref_density!(
-            Y.c.ρ[colidx],
-            p.ᶜp[colidx],
-            thermo_params,
-            surf_ref_thermo_state,
-        )
+        ᶜinterp = CCO.InterpolateF2C()
```

Comment:
This is actually only going to work for compressible, since the anelastic thermo state computation requires that pressure has already been computed, so we should put the correct elseif condition

Reply:
This way we can also error out if we're anelastic on a sphere.